Front-end/store/src/app/battle-page/battle-page.component.html

```diff
@@ -7,7 +7,7 @@
           {{battle.firstBattleBook.name}}
         </a>
       </div>
-      <img [alt]="battle.firstBattleBook.name" src="{{battle.firstBattleBook.titleImage | base64ToImgSrc}}">
+      <img [alt]="battle.firstBattleBook.name" src="{{battle.firstBattleBook.titleImage.fileUrl}}">
```

Comment:
If you moved image to separate component, you would not change source in many places.